### ISSUE TITLE
Metricbeat docker diskio module bug

### ISSUE THREAD

**Issue by kwojcicki** [elastic/beats#5568]
- Version: 
Any with docker diskio module
- Operating System: 
Any
- Steps to Reproduce: 
Start 2+ containers check the docker diskio statistics pushed. The stats will always be 0 due to a bug in the code.

There is a bug in /metricbeat/module/docker/diskio/help.go that resets the map of containers if a old stats for a container arent found.

```
func (io *BLkioService) getBlkioStats(myRawStat *docker.Stat) BlkioStats {
......
       if exist {
		myBlkioStats.reads = io.getReadPs(&oldBlkioStats, &newBlkioStats)
		myBlkioStats.writes = io.getWritePs(&oldBlkioStats, &newBlkioStats)
		myBlkioStats.totals = io.getTotalPs(&oldBlkioStats, &newBlkioStats)
	} else {
		io.BlkioSTatsPerContainer = make(map[string]BlkioRaw)
	}
```

If there are 2+ containers then the map will be reset over and over again ie:
getBlkioStats gets called for first container
old stats for first container dont exist, map reset, container1 placed into map
getBlkioStats gets called for second container
old stats for second container dont exist, map resets, container2 placed into map

rinse and repeat. I have a code fix + tests not sure if I should push into master or into a lower release and cherry pick into higher releases as well.

**Comment by andrewkroh** [comment]
@kwojcicki Thanks for reporting. Please open a PR for this. I recommend starting with a PR for master and we can backport it from there.

**Comment by kwojcicki** [comment]
Sounds good. I was having some issues pushing to the elastic beats, git was complaining about some 403 error? Probably not the best place to ask this but if you have had any experience with dealing with that issue would be helpful 😄

**Comment by andrewkroh** [comment]
You'll need to create a [fork](https://help.github.com/articles/fork-a-repo/), push the changes to your fork, then open a PR from your fork to this repo.

**Comment by joelstewart** [comment]
will this be merged into a 5.x release?

**Comment by kwojcicki** [comment]
Once PR 5582 closes I can look into making a PR for 5.5 or 5.6 shouldnt require any extra effort to back port.
